bin/metrics-vsphere.rb

```diff
@@ -0,0 +1,355 @@
+#! /usr/bin/env ruby
+#
+#   metrics-vsphere
+#
+# DESCRIPTION:
+#
+# OUTPUT:
+#   plain text, metric data, etc
+#
+# PLATFORMS:
+#   Linux, Windows, BSD, Solaris, etc
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: rbvmomi
+#
+# USAGE:
+#
+# NOTES:
+#
+# LICENSE:
+#   Yuri Zubov  <yury.zubau@gmail.com>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/metric/cli'
+require 'rbvmomi'
+
+#
+# VSphere Graphite
+#
+class VsphereGraphite < Sensu::Plugin::Metric::CLI::Graphite
+  option :host,
+         description: 'ESX or ESXi hostname',
+         short: '-H HOST',
+         long: '--host HOST'
```

Comment:
same question as before would people want to run this typically locally or external wondering if we should default to localhost.

Reply:
vSphere is hypervisor run directly on the host's hardware. we can not run it on localhost

Reply:
I figured as much but thought I would ask if you somehow got some magic way to install sensu-client on an esxi host.